### DISCUSSION TITLE
Unable to download the dependency library of the Windows version

### DISCUSSION THREAD

**Discussion by libaineu2004** [pawelsalawa/sqlitestudio#3847] [Q&A]
### Details
I downloaded the source code of the master version, ready to compile under Windows, using the VS2017 compiler. However, I found that the dependent library was missing, so I went to the website https://github.com/pawelsalawa/sqlitestudio/wiki/Instructions_for_compilation_under_Windows to download, but found that the link was invalid:
https://sqlitestudio.pl/files/depends/win32_deps.zip

Please provide binary files of third-party dependent libraries. Thank you! It is best to have both 32-bit and 64-bit.

### Operating system
Win10 x64/Qt 5.12.9 x64 + MSVC 2017 x64

### SQLiteStudio version
master

**Comment by pawelsalawa** [comment]
I'm not sure if you will manage to compile it with VS. I've tried and failed. It requires tons of code adjustments.

I'm compiling it with mingw and version 3.2.2 will be compiled with 64-bit version.

**Reply by wowocpp9527** [reply]
Hello,
I want to compile the source code with mingw
can you send me  the binary files of third-party dependent libraries. 
 It is best to have both 32-bit and 64-bit.
Thank you!
my email : wowocpp@163.com

**Reply by pawelsalawa** [reply]
I have uploaded them to https://github.com/pawelsalawa/sqlitestudio/releases/tag/3.3.2

**Reply by wowocpp9527** [reply]
thank you

**Reply by bucweat** [reply]
Hi,

I have used Qt in the past quite extensively and had very good luck getting fairly complex code to compile on SGI/MIPS (yes just dated myself), windows and linux. So from the Qt side it handles the cross platform stuff well. I'm getting ready to give this a try. I'd like to be able to run SQLiteStudio but can not bring binaries...only source code. Qt build takes a while but works and was hoping the same was true here...
